### PR TITLE
API hardening.

### PR DESCRIPTION
**Pull request by simesy**
Screen Shot 438

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -6,7 +6,9 @@
     <env name="DTT_API_URL" value="http://localhost"/>
     <env name="DTT_MINK_DRIVER_ARGS" value='["chrome", null, "http://localhost:4444/wd/hub"]'/>
     <env name="DTT_API_OPTIONS" value='{"socketTimeout": 360, "domWaitTimeout": 3600000}' />
-    <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="web/sites/default/files/browser_output"/>
+    <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="/app/web/sites/default/files/browser_output"/>
```
Comment (tannguyen04) [inline phpunit.xml:9]: We are using this file to run on CircleCI. 
`/app/web/sites/default/files/browser_output` will be not work.

Comment (simesy) [reply phpunit.xml:9]: @tannguyen04 ah yep! Were you using browser output before in CircleCI, I didn't think Circle was using it.

Comment (tannguyen04) [reply phpunit.xml:9]: we are not using output on CircleCi. But would like change it. I think We no need use absolute path for it

Comment (simesy) [reply phpunit.xml:9]: @tannguyen04 I'll do a follow up PR after this one. To work in CircleCI it will need a way to look at the result files.

**Pull request merged**